### DISCUSSION TITLE
Only getting "created" from the Survey data

### DISCUSSION THREAD

**Discussion by BlackManagarm** [rubenarslan/formr.org#563] [Q&A]
Hi,
often then i want to send an email after a survey is completed and dont get the data i need from that survey. Sometimes, without changing anything, i will get the data into the email-module but without any changes. 

Does someone got any suggestions?

**Comment by rubenarslan** [comment]
Please provide a reproducible example. If you don't specify `last(survey$field)` you'll get the first entry for that person
